artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/impl/journal/JournalStorageManager.java

```diff
@@ -163,13 +163,17 @@ protected void init(Configuration config, IOCriticalErrorListener criticalErrorL
 
       int fileSize = config.getJournalFileSize();
       // we need to correct the file size if its not a multiple of the alignement
-      int modulus = fileSize % journalFF.getAlignment();
-      if (modulus != 0) {
-         int difference = modulus;
-         int low = config.getJournalFileSize() - difference;
-         int high = low + journalFF.getAlignment();
-         fileSize = difference < journalFF.getAlignment() / 2 ? low : high;
-         ActiveMQServerLogger.LOGGER.invalidJournalFileSize(config.getJournalFileSize(), fileSize, journalFF.getAlignment());
+      if (fileSize <= journalFF.getAlignment()) {
+         fileSize = journalFF.getAlignment();
```

Comment:
I would actually throw an exception here. I wouldn't expect a block size that low. you can only have the header on the journal and nothing else if you specify fileSize == alignment.

Reply:
There is already such an exception when creating the journal (java.lang.IllegalArgumentException: File size cannot be less than 1024 bytes
    Caused by: java.lang.IllegalArgumentException: File size cannot be less than 1024 bytes") . Would setting to high be acceptable ?

Reply:
Lets not complicate things... but a unit test would be nice :)

Reply:
I will just mere it.. but if you can add an unit test...

Reply:
Done